tools/crm_mon.c

```diff
@@ -569,6 +569,24 @@ detect_user_input(GIOChannel *channel, GIOCondition condition, gpointer unused)
 }
 #endif
 
+// Basically crm_signal(SIGCHLD, SIG_IGN) plus the SA_NOCLDWAIT flag
+static void
+avoid_zombies()
+{
+    struct sigaction sa;
+
+    memset(&sa, 0, sizeof(struct sigaction));
+    if (sigemptyset(&sa.sa_mask) < 0) {
+        crm_warn("Cannot avoid zombies: %s", pcmk_strerror(errno));
+        return;
+    }
+    sa.sa_handler = SIG_IGN;
+    sa.sa_flags = SA_RESTART|SA_NOCLDWAIT;
+    if (sigaction(SIGCHLD, &sa, NULL) < 0) {
+        crm_warn("Cannot avoid zombies: %s", pcmk_strerror(errno));
+    }
```

Comment:
Using `SIGCHLD` instead of non-standardized `SIGCLD` also solved the
compilation problem under `musl` libc.